components/__tests__/Screen.tests.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import { render } from '@testing-library/react-native'
+import { Screen } from '../Screen'
+
+describe('screen template tests', () => {
```

Comment:
nit: 
```suggestion
describe('Screen component tests', () => {
```

just to be consistent but its a nit :)